src/mainboard/pcengines/apu2/romstage.c

```diff
@@ -208,9 +209,14 @@ void cache_as_ram_main(unsigned long bist, unsigned long cpu_init_detectedx)
 	bool scon = check_console();
 
 	if(scon){
+		char tmp[9];
+		strncpy(tmp,   coreboot_dmi_date+6, 4);
+		strncpy(tmp+6, coreboot_dmi_date+3, 2);
+		strncpy(tmp+4, coreboot_dmi_date,   2);
+		tmp[8] = '\0';
 		printk(BIOS_ALERT, CONFIG_MAINBOARD_SMBIOS_MANUFACTURER " "
 		                   CONFIG_MAINBOARD_PART_NUMBER "\n");
-		printk(BIOS_ALERT, "coreboot build %s\n", COREBOOT_DMI_DATE);
+		printk(BIOS_ALERT, "coreboot build %s\n", tmp);
```

Comment:
@miczyg1 please confirm if I understand that correctly, `COREBOOT_DMI_DATE` was `%Y/%m/%d` and now is `%m/%d/%Y`, tmp is `%Y/%d/%m? Why? Please add correct comment in code to.

Reply:
@pietrushnic tmp is `%Y%d%m`, without slashes.

Reply:
@krystian-hebel slashes are one thing.
@pietrushnic added correct comment. `tmp` was set to yyyymmdd incorrectly (I have mistaken the notations), should be yyyyddmm. Already fixed